src/model_config_utils.cc

```diff
@@ -2432,13 +2432,15 @@ TritonToDataType(const TRITONSERVER_DataType dtype)
 }
 
 bool
-EquivalentInNonInstanceGroupConfig(
+EquivalentInNonInstanceGroupAndNonVersionPolicyConfig(
```

Comment:
do we need to be so precise in the name - could we make it more purpose driven:

```EquivalentConfig```

```Is_Reload_Required```

Reply:
yes, the second one sounds good. It is comparing the model config between the old and the new and decides if the model version needs to be reloaded to account for the change in model config.

Reply:
updated: [Rename model config util config change require reload function](https://github.com/triton-inference-server/core/pull/388/commits/e6da04c1641636937a42ebbec71c25f81ae4876e)